pkg/trace/schema/consensus.go

```diff
@@ -81,3 +84,22 @@ func WriteBlockPart(
 		TransferTypeFieldKey:   transferType,
 	})
 }
+
+const (
+	// BlockTable is the name of the table that stores metadata about consensus blocks.
+	// following schema:
+	//
+	//  | time  | height | timestamp |
+	BlockTable = "consensus_block"
```

Comment:
perhaps?
```suggestion
	BlockTable = "consensus_block_time"
```

Reply:
👍 https://github.com/celestiaorg/celestia-core/pull/1062/commits/1d29ac2058cdfd162b643ae0a8bba2dadb4e726f

Reply:
The reason I initially avoided the `_time` suffix was that we may actually want to trace further attributes for each block. Nevertheless, I incorporated your suggestion!

Reply:
> we may actually want to trace further attributes for each block.

exactly, this was actually the same reason I think we should change it. If we want further metrics, then we will use a new table, but the name "consensus_block" is already  used by the block time

Reply:
> > we may actually want to trace further attributes for each block.
> 
> exactly, this was actually the same reason I think we should change it. If we want further metrics, then we will use a new table, but the name "consensus_block" is already used by the block time

So, does this mean that for each additional metric associated with the block, a new table will need to be created? For instance, if we intend to store the `DataHash` of the block header, would it be necessary to establish another table with a `|time|height|data_hash|` schema? Not that it is a big issue, but there seems to be potential redundancy, such as the block height being recorded in multiple tables.

Reply:
I'll defer to you on if we want to change it back. We can also change it when/if we add more things to this table.